docs/resources/css_cluster.md

```diff
@@ -83,33 +136,52 @@ The following arguments are supported:
 * `enterprise_project_id` - (Optional, String, ForceNew) Specifies the enterprise project id of the css cluster, Value 0
   indicates the default enterprise project. Changing this parameter will create a new resource.
 
-The `node_config` block supports:
+* `public_access` - (Optional, List) Specifies the public network access information.
+  The [public_access](#Css_public_access) structure is documented below.
 
-* `availability_zone` - (Optional, String, ForceNew) Availability zone (AZ). Changing this parameter will create a new
-  resource.
+* `vpcep_endpoint` - (Optional, List, ForceNew) Specifies the VPC endpoint service information.
+  The [vpcep_endpoint](#Css_vpcep_endpoint) structure is documented below.
 
-* `flavor` - (Required, String, ForceNew) Instance flavor name. For example: value range of flavor ess.spec-2u8g:
-  40 GB to 800 GB, value range of flavor ess.spec-4u16g: 40 GB to 1600 GB, value range of flavor ess.spec-8u32g: 80 GB
-  to 3200 GB, value range of flavor ess.spec-16u64g: 100 GB to 6400 GB, value range of flavor ess.spec-32u128g: 100 GB
-  to 10240 GB. Changing this parameter will create a new resource.
+* `kibana_public_access` - (Optional, List, ForceNew) Specifies Kibana public network access information.
+  This parameter is valid only when security_mode is set to true.
+  The [kibana_public_access](#Css_kibana_public_access) structure is documented below.
 
-* `network_info` - (Required, List, ForceNew) Network information. Structure is documented below. Changing this
-  parameter will create a new resource.
+* `period_unit` - (Optional, String, ForceNew) Specifies the charging period unit of the instance.
+  Valid values are *month* and *year*.
+  Changing this parameter will create a new resource.
 
-* `volume` - (Required, List, ForceNew) Information about the volume. Structure is documented below. Changing this
-  parameter will create a new resource.
+* `period` - (Optional, Int, ForceNew) Specifies the charging period of the instance.
+  If `period_unit` is set to *month*, the value ranges from 1 to 9.
+  If `period_unit` is set to *year*, the value ranges from 1 to 3.
+  Changing this parameter will create a new resource.
 
-The `network_info` block supports:
+* `auto_renew` - (Optional, String, ForceNew) Specifies whether auto renew is enabled.
+  Valid values are `true` and `false`, defaults to `false`.
+  Changing this parameter will create a new resource.
 
-* `vpc_id` - (Required, String, ForceNew) VPC ID, which is used for configuring cluster network. Changing this parameter
-  will create a new resource.
+* `auto_pay` - (Optional, String, ForceNew) Specifies whether auto pay is enabled.
```

Comment:
pls get rid of `auto_pay`, we will not support this any more

Reply:
ok,done